devenv/docker/blocks/tempo/docker-compose.yaml

```diff
@@ -1,8 +1,111 @@
+x-logging: &default-logging
+  driver: loki
+  options:
+    loki-url: 'http://localhost:3100/api/prom/push'
+    labels: namespace
+    loki-relabel-config: |
+      - action: replace
+        source_labels: ["namespace","compose_service"]
+        separator: "/"
+        target_label: job
+      - action: replace
+        source_labels: ["container_name"]
+        target_label: instance
+
+version: "3"
```

Comment:
I think the version and services line needs to be removed as the make script for devenv adds them in.

Reply:
Thanks for the chat at standup! Not the approach we want to take right now.